### ISSUE TITLE
General Comments on the User Guide

### ISSUE THREAD

**Issue by ansukert** [itc-wgtools/cPP-Tools#21]
Brad -

I did a review of the User Guide and here are my comments so far:
1. The pictures and figures are almost impossible to read - they are blurry. It may be a function of this version and will be better in the PDF, but it made the guide very hard to follow and use.
2. It may be a function of being Friday afternoon after a long week, but I didn't understand the discussion about PermaLinks in issues. I suspect it is important but I couldn't follow how to use it or what it is used for. Maybe if I look at it again it may be clearer.
3. One of the big things I didn't see how to do is to link comments to specific lines or text in a document as you can do with Comments in Word (it is one of the nicer features in Word). Maybe it is there are I just didn't recognize it.
4. I didn't see how to switch between branches so you could see what is in the 'develop' branch vs. what is in the 'Master' branch.
5. One of the good things I like is that you can enter a new Issue, go back to the document before submitting it and come back to the issue. I like this because it goes well with the way I review documents, where I make comments as I go along. The directions to create an issue were clear and easy to follow in the guide.
6. What this guide really needs for a novice like me is actual examples of how to do some of the key tasks like generating Issues. Maybe it could be in an appendix so you can include lots of screen shots without interfering with the guidance text.
7. Found a typo in Section 5.3 - "On the left side of the display is the original file and the left contains the result proposed by the Pull request" should probably be "On the left side of the display is the original file and the **right** contains the result proposed by the Pull request"
8. Section 5.5 says that you should be in 'Code View' to submit a Pull, but doesn't say how to put the document in 'Code View'.
9. In Section 5.6.1, although it is obvious you probably should explicitly say to select the draft Pull option from the drop-down menu.
10. In 5.7 there is the sentence 'As with any document with multiple editors, at some point there needs to be an agreement about what to pur in the document'. Not sure what the 'pur' is supposed to be.
11. Although the discussion of Pulls seemed OK (can't say for sure until I actually try to do it), the discussion of approving a Pull got me confused. For example, who can approve a Pull - it seems anyone can approve anyone else's Pull; not sure that is the intent. How do you select the Pull to approve. Go through that section again and see if it needs beefing up.
12. Didn't see anything about how to go to an earlier version of a document that has been approved. I can think of cases where text might have been accidentally deleted and you have to recreate it or other instances where this might be necessary. Also didn't see anything about how to view the version history of the document. 

As I try to use it I might have more comments, but hopefully these will help.

**Comment by woodbe** [comment]
@ansukert 

Thank you for the feedback. I think the images problem is that I tried to be consistent in the size, so they are being shown resized (larger or smaller), and I probably just need to drop that.

One funny thing about 2 and 3 is that they are the same. So the permalink is how you would like the comment to a specific line, so if that wasn't clear, then that will need to be cleared up. It is a messy thing in these documents, there is an easier way to get to it when GH isn't trying to render the docs (if these were C files, for example), but this is what we have, so I will work on making it more clear.

These are very helpful, and I appreciate it. I will work on them after I get through some other projects and post an update (hopefully before the end of next week).

**Comment by woodbe** [comment]
@ansukert 

Can you please take a look at this version:

https://itc-wgtools.github.io/drafts/iTC_UG_GitHub_v0.1aDRAFT.html

This is a test version I made where I removed the forced sizing on the images so they will show at max size as needed. Apparently the PDF didn't have this issue as the PDF conversion seemed to ignore that resizing for most of the images (at least that is what it seemed when I look at the PDF), but the HTML really showed the problem.

I didn't remove the resizing for images that are inline with text though, so let me know if that is still a problem (in which case I will need to do some work on changing things).

I also moved the HTML to have the TOC on the side, so let me know what you think.

I will start working on the rest later, but wanted to get your thoughts on this part immediately.

**Comment by ansukert** [comment]
Brian,

I will be om vacation the next couple of days, but I will look at this Tuesday when I get back.

Alan

**Comment by woodbe** [comment]
I have tried to take care of everything here except item 6 (not enough time).

https://itc-wgtools.github.io/drafts/iTC_UG_GitHub_v0.2DRAFT.html

I have posted the file here (if you go to https://itc-wgtools.github.io/ you can also get the PDF).
